### PR TITLE
Lesson1-3

### PR DESCRIPTION
**Pull request by tejalsd**
Created fork and student folder again.

### CODE REVIEW

**Comment by natasha-aleksandrova** [comment]
@tejalsd are these assignments that are already graded? (just wanted to confirm that this can be merged without a review)

**Comment by tejalsd** [comment]
Thanks. yes, it was resubmitted.

**Comment by natasha-aleksandrova** [comment]
@tejalsd did you mean to close this? we can just merge in what's already been graded.

**Comment by natasha-aleksandrova** [comment]
@tejalsd I just double checked and remembered that you originally did you work into an incorrect repo - so none of your work has been reviewed or graded yet.

I'm going to reopen this and do the review of your work.

File: students/Tejal/Lesson2/FizzBuzz.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+def fizzBuzz():
+    for i in range(101):
+        if (i % 3 == 0):
```
Comment (natasha-aleksandrova) [inline students/Tejal/Lesson2/FizzBuzz.py]: you don't need the parens with if statements.

File: students/Tejal/Lesson2/FizzBuzz.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+def fizzBuzz():
```
Comment (natasha-aleksandrova) [inline students/Tejal/Lesson2/FizzBuzz.py]: method names use snake_case naming convention -> `fizz_buzz`

File: students/Tejal/Lesson2/GridPrinter.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+def mlist(size):
+    list = []
```
Comment (natasha-aleksandrova) [inline students/Tejal/Lesson2/GridPrinter.py]: `list` is an internal build in function for creating a _list_

File: students/Tejal/Lesson2/series.py [hunk 4]
```diff
@@ -0,0 +1,8 @@
+def fibonacci(n):
+    if n == 0:
+        return n
+    if n == 1:
+        return 1
+    else :
+        return fibonacci(n-2) + fibonacci(n-1)
```
Comment (natasha-aleksandrova) [inline students/Tejal/Lesson2/series.py]: this is incomplete?

File: students/Tejal/Lesson3/__init__.py [hunk 5]
```diff
@@ -0,0 +1,5 @@
+
+import Slicing
+
+a_string = "this is a string"
+Slicing.exchange_first_last(a_string)
```
Comment (natasha-aleksandrova) [inline students/Tejal/Lesson3/__init__.py]: also looks incomplete

File: students/Tejal/Lesson3/mailroom.py [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+import time
+
+
+
+donations = {1:{'name':'John Kestrel','amount':[653772.32, 12.17, 4]},
+2:{'name':'Jeff Kingbird','amount':[877.33]},
+3:{'name':'Paul Jacobin','amount':[663.23, 43.87, 1.32]},
+4:{'name':'Mark Tanager','amount':[1663.23, 4300.87, 10432.0]},
+5:{'name':'Anna Hummingbird','amount':[4500.0, 800.0, 2350.0]},
+6:{'name':'Calvin Fannin', 'amount':[645.0]}}
```
Comment (natasha-aleksandrova) [inline students/Tejal/Lesson3/mailroom.py]: you should be using a list for your data structure in this lesson - please correct.

**Pull request merged**